Bottom: e9163142953e01c15c3e0cba9bd865f04f52d9dc
Top:    56ef8862a6a93a9854916b06f89873c9e093ae89
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:13 +0000

Reduce tolerances a smidgin


---

diff --git a/src/netclass.c b/src/netclass.c
index 54c6d6e..6003f1b 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
-    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3), MM_TO_COORD (4)},
+    {MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (4), MM_TO_COORD (0.3)}
   };
